test/unit/plugin/v2/api_reporter_test.rb

```diff
@@ -0,0 +1,39 @@
+require "helper"
+
+require "inspec/plugin/v2"
+
+describe "Reporter plugin type" do
+  describe "when registering the plugin type superclass" do
+    it "returns the superclass when calling the global defintion method" do
+      klass = Inspec.plugin(2, :reporter)
+      _(klass).must_be_kind_of Class
+      _(klass).must_equal Inspec::Plugin::V2::PluginType::Reporter
+    end
+
+    it "returns the superclass when referenced by alias" do
```

Comment:
I'm maybe not thinking in Ruby here, but isn't `Inspec::Plugin::V2::PluginBase` the superclass and `Inspec::Plugin::V2::PluginType::Reporter` the subclass?

Reply:
"The superclass" here meaning "the superclass we are creating for the new plugin type" - that is, the class under test at line 6.

Reply:
This example says that it `returns the superclass` but the assertion is `_(klass).must_equal Inspec::Plugin::V2::PluginType::Reporter`

In inheritance terminology (e.g. http://rubylearning.com/satishtalim/ruby_inheritance.html ) `Inspec::Plugin::V2::PluginType::Reporter` is the subclass not the superclass